specs/integrations/synthetics.md

```diff
@@ -0,0 +1,119 @@
+## Synthetics Integration
+
+Synthetic monitors play a crucial role in periodically checking the status of your services and applications on a global scale. General documentation about synthetic monitors can be found in
+[Synthetics getting started page](https://www.elastic.co/guide/en/observability/current/synthetics-get-started.html).
+
+This integration goes in to more detail about how the sythetics monitors would
+be correlated with the APM traces. Synthetics traces can be categorized in to two
+main types
+  1. HTTP checks - These have one-one mapping with APM transactions
+  2. Browser checks - These have a one-to-many mapping with APM transactions
+
+### Correlation
```

Comment:
@felixbarny @sqren Looking for some feedback here, We previously had plans to integrate APM-Synthetics using the [`service.name`](https://www.elastic.co/guide/en/beats/heartbeat/current/monitor-options.html#service-name) route where we would look for services from APM and link directly to the Service overview in APM. This does not work right now, I am not sure if anyone is using it. 

Do you think its still a valid use-case with this new proposal where we the integration point is based on the URL's being traced instead of `service` monitored via APM. 

Example:

```
npx @elastic/synthetics --apm_tracing_urls "elastic.co/*"
```

Reply:
I think that's just a very different type of correlation. It won't allow us to link from the synthetics network tab for a monitor execution to a trace waterfall and vice-versa. That connection mostly makes sense for services that are monitored with the RUM agent. We could even suggest the creation of a synthetic monitor when we know it's monitored by RUM.

I think the correlation that's proposed here and correlation by service.name can complement each other very well. This is for linking monitor executions to traces. The service.name correlation would allow us to show synthetic monitors in the context of a (RUM) service.

Reply:
>The service.name correlation would allow us to show synthetic monitors in the context of a (RUM) service. 

Thats a great point. Yeah as you mentioned they should complement each other.

I brought this topic to discuss this scenario. Lets say the synthetics monitors ends up creating `N amount of transactions in the APM backends`, What would be the best landing page for the user to go from Synthetics Step Overview in the context of APM, Is it the `Service Overview Page` with that `trace.id` pre applied or Linking to all traces that are originating from `Synthetics monitors`. 

This is in addition to the individual network request -> APM transaction link.